app/graphql/types/need_type.rb

```diff
@@ -0,0 +1,18 @@
+module Types
+  class NeedType < Types::BaseObject
+    field :id, ID, null: false
+    field :point_of_contact, String, null: false
+    field :title, String, null: false
+    field :description, String, null: false
+    field :start_time, String, null: false
+    field :end_time, String, null: false
```

Comment:
there might be times when a start and end time don't make sense and we want to keep them nil